lib/gcloud/resource_manager/project/updater.rb

```diff
@@ -0,0 +1,127 @@
+#--
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "time"
+require "gcloud/resource_manager/errors"
+
+module Gcloud
+  module ResourceManager
+    class Project
+      ##
+      # = Project Updater
```

Comment:
I think it would be good to adapt the class-level description for Dns::Zone::Transaction to this class, same idea...